pkg/credentials/aws/aws.go

```diff
@@ -332,6 +350,7 @@ func (a *AuthSAML) Retrieve() (*Credentials, error) {
 	}
 
 	return &Credentials{
+		AwsProfile:      "default",
```

Comment:
This should probably be a constant somewhere. `pkg/config/constant/api.go` is a good place if its needed globally